### PR TITLE
Reg docs

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Updates and fixes issues with the docs for the registry salt util, execution module, and state module.
Fixes formatting issues

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/pull/47107#issuecomment-382964960

### Tests written?
NA

### Commits signed with GPG?
Yes

### CODE REVIEW

**Comment by hrumph** [comment]
Thanks @Twangboy. In some places you put the param type for vdata as __str, int, list__ but shouldn't it really be __str, int, list, bytes__?

**Review by cachedout — COMMENTED** [review]
I really think a lot of this documentation could do with some examples to make it easier to understand.

**Comment by hrumph** [comment]
Typing from phone so please forgive crappy English+spelling. I think the pr
should be accepted even if more examples are merited ( I'll stay out of
that argument). @twangboy s documentation pr is a vast improvement over the
status quo so should be accepted on that basis

On Apr 23, 2018 1:11 PM, "Mike Place" <notifications@github.com> wrote:

*@cachedout* commented on this pull request.

I really think a lot of this documentation could do with some examples to
make it easier to understand.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/saltstack/salt/pull/47216#pullrequestreview-114475296>,
or mute the thread

**Comment by rallytime** [comment]
@twangboy There's some lint errors on this now. Can you take another look?

https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/21364/violations/

File: salt/modules/reg.py [hunk 1]
```diff
@@ -76,13 +118,18 @@ def broadcast_change():
     '''
     Refresh the windows environment.
 
-    Returns (bool): True if successful, otherwise False
+    .. note::
+        This will only effect new processes and windows. Services will not see
+        the change until the system restarts.
```
Comment (damon-atkins) [inline salt/modules/reg.py]: until the service is restarted.

Comment (twangboy) [reply salt/modules/reg.py]: @damon-atkins Any services running under the Local System account will not receive the new environment until you restart.
https://support.microsoft.com/en-us/help/821761/changes-that-you-make-to-environment-variables-do-not-affect-services

Comment (damon-atkins) [reply salt/modules/reg.py]: un-expected, but makes senses,

**Comment by twangboy** [comment]
Rebased

**Comment by rallytime** [comment]
@twangboy There's still some lint failing here: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/21454/violations/file/salt/states/reg.py/

**Comment by hrumph** [comment]
@Twangboy I still think that where it says _vdata (str, int, list)_, it should really say
_vdata (str, int, list, bytes)_.

**Comment by twangboy** [comment]
@rallytime @hrumph Done

**Comment by hrumph** [comment]
Thanks for the work @twangboy

**Review by rallytime — APPROVED** [review]
👍

**Pull request merged**